packages/scripts/webapp/preset/config/webpack.config.js

```diff
@@ -121,11 +118,80 @@ function getVersions() {
 	};
 }
 
+function getJSLoaders(angularLegacy) {
+	const loaders = [
+		{ loader: 'cache-loader' },
+		{
+			loader: 'babel-loader',
+			options: getBabelConfig(),
+		},
+	];
+
+	if (angularLegacy) {
+		loaders.unshift({
+			loader: 'ng-annotate-loader',
+			options: {
+				ngAnnotate: 'ng-annotate-patched',
+				es6: true,
+				explicitOnly: false,
+			},
+		});
+	}
+
+	return loaders;
+}
+
+function getHTMLLoaders(angularLegacy) {
+	const loaders = [
+		{ loader: 'cache-loader' },
+		{
+			loader: 'html-loader',
+			options: {
+				minimize: true,
+				removeComments: true,
+				collapseWhitespace: true,
+			},
+		},
+	];
+
+	if (angularLegacy) {
+		loaders.unshift({ loader: 'ngtemplate-loader' });
+	}
+
+	return loaders;
+}
+
+function getCopyConfig(userCopyConfig = []) {
+	const config = [...userCopyConfig];
+	const assetsOverridden = config.some(({ from }) => from === 'src/assets');
```

Comment:
if `from` is equals to `to`, you can pass an array https://github.com/webpack-contrib/copy-webpack-plugin#from

Reply:
Changed to check if we have an object or not